dynatrace.tf

```diff
@@ -134,6 +134,11 @@ resource "helm_release" "dynatrace_operator" {
     value = "${var.dynatrace_api}/v1/deployment/installer/agent/unix/default/latest?arch=x86"
   }
 
+  set {
+    name  = "classicFullStack.args[0]"
+    value = "--set-host-group=${var.environment}_CRIME_CP_AKS"
```

Comment:
Maybe use ${upper(var.environment)} if the host group is case sensitive.
Also nft environment variables will be nft01 and nft02 but I belive they use same host group might need to check.

Reply:
@Abhivan even though it works (DT confirmed) I agree it is better to upper case to keep it consistent with the rest of the variables. I pushed your change